_board/waveshare_rp2040_geek.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: download
+board_id: "waveshare_rp2040_geek"
+title: "RP2040-GEEK Download"
+name: "RP2040-GEEK"
+manufacturer: "Waveshare"
+board_url:
+ - "https://www.waveshare.com/rp2040-geek.htm"
+board_image: "waveshare_rp2040_geek.jpg"
+date_added: 2024-03-29
+family: raspberrypi
+features:
+  - USB-C
```

Comment:
AFAIK none of these boards (esp32s3 and rp2040) have USB-C

Reply:
Ugh, I don't know where my head was, good catch, I'll submit a new PR to drop the RP2040 WiFi and remove USB-C from both the RP2040 and S3 Geek modules.

Thanks!